src/coreclr/vm/i386/asmhelpers.asm

```diff
@@ -1432,6 +1412,22 @@ public _ThePreStubPatchLabel@0
     ret
 _ThePreStubPatch@0 endp
 
+_TheUMEntryPrestub@0 proc public
+    ; push argument registers
+    push        ecx
```

Comment:
Are only `ecx` and `edx` pushed/saved because all other possible arguments on the stack?

Reply:
Yup.